defaults.env

```diff
@@ -0,0 +1,22 @@
+# DEFAULTS.ENV is a Docker .env file for configuring cores aspects of Janeway
```

Comment:
Do the .env files need to be in root?

Reply:
I thought they did, but, it appears you can use a path, here are the docs: https://docs.lando.dev/config/env.html#environment-files

Would you prefer a folder for these kinds of things? If so, if there's an existing folder I should use (I didn't see anything immediately obvious) let me know and I'll move them. Likewise, if we need a new folder, what should we call it? Since we have a 'src' folder, maybe 'etc'?

Reply:
moved to new etc folder